src/langs/assemblyscript/package.json

```diff
@@ -13,6 +13,6 @@
     "url": "https://github.com/boyanio/wasm-wheel.git"
   },
   "devDependencies": {
-    "assemblyscript": "github:AssemblyScript/assemblyscript#918a40e"
+    "assemblyscript": "github:AssemblyScript/assemblyscript"
```

Comment:
I had put on purpose the commit hash, because the master branch could change with breaking changes and thus breaking the compilation. How do you think we should proceed in this scenario?

Reply:
used in this period version already pinned in [`package-lock.json`](https://github.com/boyanio/wasm-wheel/pull/10/files#diff-85c1bc23d8c830f5bb0ec479a5e1bf7fR14)